src/help/guides/extension-development/function-definitions.md

```diff
@@ -1,13 +1,24 @@
 # Required Methods
+
 ## getMangaDetails
+
+### Method signature
+
 `async getMangaDetails(mangaId: string): Promise<Manga>`
 
-- A manga ID is defined by the source developer. However IDs are provided to Paperback in other functions, will be given to this function in the same format.
+### Parameters
 
-Given an ID, this async function should return a filled out Manga object.
+| Parameter | Type | Description|
+|-----------|------|------------|
+|  `mangaId`  | String | The ID of a manga. The manga ID is provided from the other discovery functions, such as the home page and performed searches. The manga ID should be used to return information about the manga, such as the title and author.|
```

Comment:
Be specific with the function names. Even better, link them!

Reply:
The functions aren't added, but I will link them once I do add them in.